src/index.ts

```diff
@@ -498,6 +498,9 @@ class QueryPipeline extends EventEmitter {
           jsonProto.searchResponse?.totalResults || "0",
           10
         ),
+        ...(jsonProto.banners && {
+          banners: jsonProto.banners,
+        }),
```

Comment:
Can it be simplified like this?

```suggestion
        banners: jsonProto?.banners,
```